plutus-use-cases/src/Plutus/Contracts/GameStateMachine.hs

```diff
@@ -13,7 +13,8 @@
 {-# LANGUAGE TypeOperators         #-}
 {-# LANGUAGE ViewPatterns          #-}
 {-# OPTIONS_GHC -fno-ignore-interface-pragmas #-}
-{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:debug-context #-}
+{-# OPTIONS_GHC -g -fplugin-opt PlutusTx.Plugin:coverage-all #-}
+-- You need to use all of these to get coverage
```

Comment:
```suggestion
-- You need to use all of these to get coverage

```

I think you need an extra empty line here to not break the haddock comment below it.